### PR TITLE
Add Evan's updates 02/24/2017

### PR DESCRIPTION
**Pull request by evanlimanto**

### CODE REVIEW

File: updates/Evan Limanto.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Evan Limanto - weekly updates
+
+## 02-24-2017
+
+- **Past week:** Worked on 3 bug fix PRs, all tests passing and awaiting review
```
Comment (stefanv) [inline updates/Evan Limanto.md]: Please add the URLs here.  This document is really meant as a  kind of log book that we can refer back to at the end of the program.

**Comment by evanlimanto** [comment]
Done.

**Comment by stefanv** [comment]
Sorry for being a PITA, but I still can't click on those links :/

https://daringfireball.net/projects/markdown/syntax

**Comment by evanlimanto** [comment]
Oops sorry, thought you meant just issue numbers. Fixed and tested.

**Comment by stefanv** [comment]
Perfect, thanks!

**Pull request merged**